doc/usermanual/darkroom/modules/correction/denoise_profiled.xml

```diff
@@ -115,11 +137,34 @@
       <title>search radius</title>
       <para>
         This slider is only available if mode <quote>non-local means</quote> is selected. It
-        controls the how far from a pixel the algorithm will try to find similar patches. The
+        controls how far from a pixel the algorithm will try to find similar patches. The
         processing time is hugely impacted by this parameter: it depends on the square of the
         parameter. A lower value will make execution faster, a higher value will make it slower.
         Increasing the value can give better results for very noisy images when coarse grain
-        noise is visible.
+        noise is visible, but you should better use the scattering slider instead.
```

Comment:
sentence could placed after: ... try to find similar patches. Increasing the value ...